recipes/qt4/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set version = "4.8.7" %}
+
+package:
+  name: qt
+  version: {{ version }}
+
+source:
+  fn: qt-everywhere-opensource-src-{{ version }}.tar.gz
+  url: http://download.qt-project.org/official_releases/qt/4.8/{{ version }}/qt-everywhere-opensource-src-{{ version }}.tar.gz
+  md5: d990ee66bf7ab0c785589776f35ba6ad
+  patches:
```

Comment:
Definitely interested in each of these patches either having good comments, or a xref to a patch to the repo.